tests/functional/settings_secondary_emails.js

```diff
@@ -317,6 +320,28 @@ define([
         .then(switchToWindow(1))
         .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER))
         .then(closeCurrentWindow());
-    }
+    },
+    'add secondary email, hit Esc, input should be cleared': function () {
+      const TEXT = 'TEST';
+      return this.remote
+            // create user, verify, sign in, open settings page
```

Comment:
@ mutdmour could you fix indentation in this test, compare to what you see above

Reply:
@mutdmour ^